src/screens/WeatherCardScreen/WeatherCardScreen.tsx

```diff
@@ -28,51 +32,31 @@ export const WeatherCardScreen = (
   >,
 ) => {
   const { route, navigation } = props;
-  const { title } = route.params!;
+  const { title } = route.params;
 
   const dispatch = useDispatch();
-  const statusApp = useSelector(selectStatusApp);
 
-  const currentCity = useSelector<AppStoreType, DataItemType[]>((state) =>
-    state.cities.cities.filter((city) => city.city === title),
-  );
+  const statusApp = useSelector(selectStatusApp);
+  const cities = useSelector(getCities);
+  const dataFromRedux = useSelector(getDayWeatherInfo);
+  const error = useSelector(getError);
 
-  const [data, setData] = useState<dayWeatherInfo | null>(null);
-  const [error, setError] = useState<boolean>(false);
+  const currentCity = cities.find((city) => {
+    if (title && city.city === title) {
+      return city;
+    }
+  });
```

Comment:
if you won't find any city, it will returns `undefined`. Add mock value for this case